src/Pest.php

```diff
@@ -8,3 +8,8 @@ function version(): string
 {
     return '1.0.4';
 }
+
+function testDirectory(string $file = ''): string
+{
+    return $_ENV['PEST_TEST_DIRECTORY'] . $file;
```

Comment:
Replace this by `$_SERVER`.

Reply:
Just this occurrence or all `$_ENV` usages in this PR?

Reply:
Can you actually use the `TestSuite::getInstance()->setTestDirectory()` instead of using ENVs?

Reply:
Let me test but if I remember right it wasn't possible because the test path had to be set before the instance was created.

Reply:
Yeah that doesn't work because https://github.com/pestphp/pest/blob/master/src/TestSuite.php#L98-L100 ends up executing some things that already need access to the test path.

Reply:
An alternative could be this change.

```php
$testSuite = TestSuite::getInstance($rootPath, (new ArgvInput())->getParameterOption('--test-directory', 'tests'));

public static function getInstance(string $rootPath = null, string $testPath = null): TestSuite
{
    if (is_string($rootPath)) {
        self::$instance = new TestSuite($rootPath);

        if (is_string($testPath)) {
            self::$instance->testPath = $testPath;
        }

        foreach (Plugin::$callables as $callable) {
            $callable();
        }

        return self::$instance;
    }

    if (self::$instance === null) {
        throw new InvalidPestCommand();
    }

    return self::$instance;
}
```

Reply:
Alternative https://github.com/faustbrian/pest/commit/c8a3e7e1f4747f838292b75e44be8ac33892f2ff but results in an odd issue I need to check if this is preferred.

```
❯ bin/pest --test-directory=tests
PHPUnit 9.5.3 by Sebastian Bergmann and contributors.

Unknown option "--test-directory"
```